packages/llvm.rb

```diff
@@ -27,10 +27,13 @@ class Llvm < Package
   depends_on 'py3_pygments' => :build
   depends_on 'ccache' => :build
   depends_on 'elfutils' # R
-  depends_on 'gcc' # R
+  depends_on 'gcc11::libs' # R
```

Comment:
If we're going to include version specific packages, we need to make sure if gcc is updated to version 12, these dependency lines are updated too.

Reply:
We need to figure out how to make this work with `gcc::libs` so we don't have to specify the version number.

Reply:
This is no longer an issue if we rename gcc12.rb to gcc.rb and remove the version?
```
depends_on 'gcc::libs'
```

Reply:
Are we still trying to abstract the gcc version?